src/api-handlers/candidates.js

```diff
@@ -90,12 +91,61 @@ async function getCandidates(fastify, request) {
 
 }
 
+async function getPlanetCandidates(fastify, request) {
+    const {
+        query: { walletId },
+        params: { dacId },
+    } = request;
+    const config = loadConfig();
+    const logger = require('../connections/logger')('candidates', config.logger);
+    const dacConfig = await loadDacConfig(fastify, dacId);
+    const api = new Api({
+        rpc: new JsonRpc(config.eos.endpoint, {fetch}),
+        signatureProvider: null,
+        chainId: config.chainId,
+        textDecoder: new TextDecoder(),
+        textEncoder: new TextEncoder(),
+    });
+    const db = fastify.mongo.db;
+    //
+    const candidates = await getCandidates(logger, api, dacId, 100);
```

Comment:
Probably we should define the limit argument somewhere to avoid a "_magic number_" and then pass the defined constant. i.e. `const limit = 100`